css/style.scss

```
@@ -10,6 +10,7 @@
 	margin-bottom: 5px;
 	margin-left: 5px;
 	text-overflow: ellipsis;
+	word-break: break-word;
```

```suggestion
	word-break: normal;
	overflow-wrap: anywhere;
```
Seems `word-break: break-word;` is deprecated https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#values

---

done